connectivity/check/test.go

```diff
@@ -218,8 +223,8 @@ func (t *Test) WithPolicy(policy string) *Test {
 	for i := range pl {
 		pl[i].Namespace = t.ctx.params.TestNamespace
 		if pl[i].Spec != nil {
-			// Check both 'io.kubernetes.pod.namespace' and 'k8s:io.kubernetes.pod.namespace'.
-			for _, k := range []string{k8sConst.PodNamespaceLabel, kubernetesSourcedLabelPrefix + k8sConst.PodNamespaceLabel} {
+			// Check both 'io.kubernetes.pod.namespace' and 'k8s:io.kubernetes.pod.namespace' and 'any:io.kubernetes.pod.namespace'.
```

Comment:
The "both" in the sentence is no longer correct 😄 Let's just drop the comment altogether, it doesn't add any additional information.